### PR TITLE
Add 2 examples to GL JS quickstart + consistify code, code comments

### PR DESCRIPTION
**Pull request by domlet**
Changes to make GL JS quickstart and examples more accessible to beginners.

Intent is to reduce negative docs user feedback ratings from code newbies and welcome more developers to build with Mapbox tools. :)

- consistify order of `map` parameters in code
- consistify code comments, especially "stylesheet location" --> "style URL"
- add two links to GL JS examples after quickstart instructions
- clarify that GL JS 'add a map' examples are adding a map "to a webpage" 🤷 

Tagging @HeyStenson @ryanhamley for review.

Screenshot:

### CODE REVIEW

**Review by ryanhamley — APPROVED** [review]

File: docs/components/quickstart.js [hunk 1]
```diff
@@ -13,8 +13,8 @@ class Quickstart extends React.Component {
     }
 
     mapOptions = `{
-    container: 'map',
-    style: 'mapbox://styles/mapbox/streets-v11', // stylesheet location
+    container: 'map', // container id
```
Comment (HeyStenson) [inline docs/components/quickstart.js:16]:
```suggestion
    container: 'map', // container ID
```

File: docs/pages/api/index.md [hunk 2]
```diff
@@ -53,6 +52,9 @@ To get started, you need an [access token](https://docs.mapbox.com/help/how-mapb
 <Quickstart />
 }}
 
+You can see and interact with similar code in Mapbox GL JS examples like these:
```
Comment (HeyStenson) [inline docs/pages/api/index.md:55]:
```suggestion
You can see and interact with similar code in these Mapbox GL JS examples:
```

❓

File: docs/pages/api/index.md [hunk 3]
```diff
@@ -140,8 +142,8 @@ import MapboxWorker from 'worker-loader!mapbox-gl/dist/mapbox-gl-csp-worker'; //
 
 mapboxgl.workerClass = MapboxWorker; // Wire up loaded worker to be used instead of the default
 let map = new mapboxgl.Map({
-    container: 'map',
-    style: 'mapbox://styles/mapbox/streets-v11', // stylesheet location
+    container: 'map', // container id
+     style: 'mapbox://styles/mapbox/streets-v11', // styleURL
```
Comment (HeyStenson) [inline docs/pages/api/index.md:146]:
```suggestion
    container: 'map', // container ID
     style: 'mapbox://styles/mapbox/streets-v11', // style URL
```

File: docs/pages/api/index.md [hunk 4]
```diff
@@ -170,8 +172,8 @@ import MapboxWorker from 'mapbox-gl/dist/mapbox-gl-csp-worker';
 
 mapboxgl.workerClass = MapboxWorker;
 let map = new mapboxgl.Map({
-    container: 'map',
-    style: 'mapbox://styles/mapbox/streets-v11', // stylesheet location
+    container: 'map', // container id
+     style: 'mapbox://styles/mapbox/streets-v11', // style URL
```
Comment (HeyStenson) [inline docs/pages/api/index.md:176]:
```suggestion
    container: 'map', // container ID
    style: 'mapbox://styles/mapbox/streets-v11', // style URL
```

File: docs/pages/example/change-case-of-labels.html [hunk 5]
```diff
@@ -2,7 +2,7 @@
 <script>
     var map = new mapboxgl.Map({
         container: 'map', // container id
```
Comment (HeyStenson) [inline docs/pages/example/change-case-of-labels.html:4]:
```suggestion
        container: 'map', // container ID
```

File: docs/pages/example/display-and-style-rich-text-labels.html [hunk 6]
```diff
@@ -6,7 +6,7 @@
 
     var map = new mapboxgl.Map({
         container: 'map', // container id
```
Comment (HeyStenson) [inline docs/pages/example/display-and-style-rich-text-labels.html:8]:
```suggestion
        container: 'map', // container ID
```

File: docs/pages/example/render-world-copies.html [hunk 7]
```diff
@@ -23,7 +23,7 @@
 <script>
     var map = new mapboxgl.Map({
         container: 'map', // container id
```
Comment (HeyStenson) [inline docs/pages/example/render-world-copies.html:25]:
```suggestion
        container: 'map', // container ID
```

File: docs/pages/example/satellite-map.html [hunk 8]
```diff
@@ -1,9 +1,9 @@
 <div id="map"></div>
 <script>
     var map = new mapboxgl.Map({
-        container: 'map',
-        zoom: 9,
-        center: [137.9150899566626, 36.25956997955441],
-        style: 'mapbox://styles/mapbox/satellite-v9'
+        container: 'map', // container id
```
Comment (HeyStenson) [inline docs/pages/example/satellite-map.html:4]:
```suggestion
        container: 'map', // container ID
```

File: docs/pages/example/visualize-population-density.html [hunk 9]
```diff
@@ -2,7 +2,7 @@
 <script>
     var map = new mapboxgl.Map({
         container: 'map', // container id
```
Comment (HeyStenson) [inline docs/pages/example/visualize-population-density.html:4]:
```suggestion
        container: 'map', // container ID
```

**Review by HeyStenson — APPROVED** [review]
This is beaut @domlet! 🍊

**Pull request merged**